lib/definitions.php

```diff
@@ -279,6 +279,25 @@
 define('RTMESSAGE_SERVICE_CHANGE', 32768);
 define('RTMESSAGE_TYPE_CHANGE', 65536);
 
+define('NETWORK_INTERFACE_TYPE_ACCESS', 0);
+define('NETWORK_INTERFACE_TYPE_AGGREGATION', 1);
+define('NETWORK_INTERFACE_TYPE_BACKBONE', 2);
+
+$NETWORK_INTERFACE_TYPES = array(
+	NETWORK_INTERFACE_TYPE_ACCESS => array(
+		'label' => trans('Access'),
+		'name' => 'INTERFACE_TYPE_ACCESS'
+	),
+	NETWORK_INTERFACE_TYPE_AGGREGATION => array(
+		'label' => trans('Aggregation'),
+		'name' => 'INTERFACE_TYPE_AGGREGATION'
+	),
+	NETWORK_INTERFACE_TYPE_BACKBONE => array(
+		'label' => trans('Backbone'),
+		'name' => 'INTERFACE_TYPE_BACKBONE'
+	)
+);
+
 //Request Tracker Ticket Types
```

Comment:
Po co wydzielasz warstwy sieci? Moim zdaniem to zbyteczne.
Wytłumacz proszę.

Reply:
Nie robimy na tą chwilę nowych obiektów "interfejs sieciowy" bo to nie jest przedmiotem zgłoszenia a ten temat jest bardzo szeroki. Ale potrzebna nam zdefiniowanie tej właściwości.

W przyszłości będzie można ją skopiować wyżej gdy powstanie nowy obiekt interjfejs sieciowy którego któreś tam piętro w drzewku to "sektor radiowy" czyli interfejs bezprzewodowy o typie dostępowy.

Reply:
Ale po co gromadzić w bazie informację o jednej z 3 warstw? Bez sensu to jest - UKE sobie wymyśliło i wszystkie powielają. Od strony technicznej zawsze były interfejsy UNI i NNI.

Reply:
Zawsze były NNI i UNI - ale potem gdzieś się spotkałem z tym że NNI zaczeli dzielić bardziej na: agregacje, backbone oraz upstream.

Dla mnie mogę mieć sektory o typach NNI i UNI. Nie widzę problemu, jest to wystarczające - przerobię commita.

Reply:
Coś w ten deseń? Może zasugerujesz jakieś lepsze krótsze tłumaczenia?